static/app/components/replays/replayView.tsx

```diff
@@ -50,4 +78,23 @@ const ContextContainer = styled('div')`
   gap: ${space(1)};
 `;
 
+const PlayerContainer = styled('div')`
+  display: grid;
+  grid-auto-flow: row;
+  grid-template-rows: auto 1fr;
+  gap: ${space(1)};
+  width: 100%;
```

Comment:
I think this is taking up too much space, the sidebar isn't getting 25% on my screen. 

Instead this should do it because this is a child of PlayerBreadcrumbContainer, which is a flex parent.
```suggestion
  flex-grow: 1;
```